### PR TITLE
dev-db/percona-xtrabackup: bump to 8.0.7

### PR DESCRIPTION
**Pull request by hydrapolic**
Builds pass on ~amd64. 

Thank you.

### CODE REVIEW

File: dev-db/percona-xtrabackup/percona-xtrabackup-8.0.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+RDEPEND="
+	${DEPEND}
+	!dev-db/percona-xtrabackup-bin
+	dev-perl/DBD-mysql"
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_CONFIG=xtrabackup_release
+		-DBUILD_SHARED_LIBS=OFF
```
Comment (juippis) [inline dev-db/percona-xtrabackup/percona-xtrabackup-8.0.7.ebuild]: This has probably been over already, but could you tell me why this is so? Boost-related?

File: dev-db/percona-xtrabackup/percona-xtrabackup-8.0.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	dev-libs/libevent:0=
+	dev-libs/libgcrypt:0=
+	dev-libs/libgpg-error
+	dev-libs/openssl:0=
+	dev-libs/protobuf:=
+	dev-libs/rapidjson
+	dev-libs/re2:=
+	dev-python/sphinx
+	net-misc/curl
+	sys-libs/zlib:="
```
Comment (juippis) [inline dev-db/percona-xtrabackup/percona-xtrabackup-8.0.7.ebuild]: Reminder that this was a mistake and you shouldn't use subslot binder on zlib :)

Comment (hydrapolic) [reply dev-db/percona-xtrabackup/percona-xtrabackup-8.0.7.ebuild]: The binary built fine, but was unusable. It was a tip by @mgorny

File: dev-db/percona-xtrabackup/percona-xtrabackup-8.0.7.ebuild (hunk 1 above)
Comment (hydrapolic) [reply dev-db/percona-xtrabackup/percona-xtrabackup-8.0.7.ebuild]: The binary was built without it but was unusable (runtime errors)

**Comment by hydrapolic** [comment]
Thanks @juippis

**Pull request closed without merging**